student.rb

```diff
@@ -6,6 +6,7 @@ class Student < Person
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
- [ ] @SanaTameem great work on your student class, in order to achieve the `belongs-to' side of the student and classroom many-to-many relationship, you need to add a new method classroom to achieve this instead of adding it in the initialize method eg
```
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
- Well resolved 👍🏽